docs/reference/errors-and-warnings/NU5046.md

```diff
@@ -0,0 +1,58 @@
+---
+title: NuGet Error NU5046
+description: NU5046 Error code
+author: dominoFire
+ms.author: feaguila
+ms.date: 9/4/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5046
+---
+
+# NuGet Error NU5046
+
+<pre>The icon file 'Icon.png' does not exist in the package.</pre>
+
+
+# Issue
+
+NuGet is unable find the local file that is marked as the package icon.
+
+
+# Solution
+
+- Make sure that the file that is marked as the package icon exists and it is readable.
```

Comment:
Again, this sentence makes it sound like it's looking for the file on the filesystem, not in the package. I also find the term "marked" a bit strange and would prefer if the sentence could be changed to avoid it.

In fact, is this point even needed? If the file doesn't exist on the local file system, but the nuspec has a `file` element, or the csproj has a `<None Include="icon.jpg" />` element, but the file doesn't exist on the filesystem (or isn't readable), wouldn't pack return a different error code? Unreadable file trying to be included in the package isn't icon specific.